pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -137,39 +137,39 @@ func TestEnoughRequests(t *testing.T) {
 				newResourcePod(framework.Resource{MilliCPU: 10, Memory: 20})),
 			name:                      "too many resources fails",
 			wantStatus:                framework.NewStatus(framework.Unschedulable, getErrReason(v1.ResourceCPU), getErrReason(v1.ResourceMemory)),
-			wantInsufficientResources: []InsufficientResource{{v1.ResourceCPU, getErrReason(v1.ResourceCPU), 1, 10, 10}, {v1.ResourceMemory, getErrReason(v1.ResourceMemory), 1, 20, 20}},
+			wantInsufficientResources: []InsufficientResource{{ResourceName: v1.ResourceCPU, Reason: getErrReason(v1.ResourceCPU), Requested: 1, Used: 10, Capacity: 10}, {ResourceName: v1.ResourceMemory, Reason: getErrReason(v1.ResourceMemory), Requested: 1, Used: 20, Capacity: 20}},
```

Comment:
Thank you for adding the field names :)

Could you also split it in several lines? Something like:

```golang
[]InsufficientResource{
  {ResourceName: v1.ResrouceCPU, Reason: ...},
  {ResourceName: v1.ResourceMemory, Reason: ..},
}
```

Reply:
done